tests/src/test/java/org/eclipse/hono/tests/coap/CoapTestBase.java

```diff
@@ -125,6 +129,9 @@ public abstract class CoapTestBase {
     private static final String COMMAND_TO_SEND = "setDarkness";
     private static final String COMMAND_JSON_KEY = "darkness";
 
+    // TODO: possibly add a dependency on the `hono-adapter-coap` module and use the constant from there?
```

Comment:
definitely

Reply:
Having problems defining the dependency in `hono-tests` like I mentioned on the gitter chat.

> I just tried adding a dependency in the hono-tests pom.xml to the hono-adapter-coap module but for some reason, that module is unable to derive the version and starts looking for org.eclipse.hono:hono-adapter-coap:jar:unknown in https://jcenter.bintray.com/
>
>Any idea what I'm doing wrong here? I've tried to search for some other/more things to specify but other "project dependencies" don't seem to require anything else. The hono-core and hono-service-device-registry-base dependencies don't specify any version explicitly at least.

and 

>the difference I see is that hono-service-device-registry-base and hono-core are explicitly stated in the bom while hono-adapter-coap is not (only hono-adapter-base is specified in the bom). Maybe that has something to do with this... but it doesn't really tell me what the correct way to fix this would be.

Reply:
This has been resolved.